spec/features/admin_claim_check_spec.rb

```diff
@@ -36,14 +39,51 @@
       end
     end
 
+    scenario "they can reject a claim" do
+      submitted_claims = create_list(:claim, 5, :submitted)
+      claim_to_reject = submitted_claims.first
+
+      click_on "View claims"
+
+      expect(page).to have_content(claim_to_reject.reference)
+      expect(page).to have_content("5 claims awaiting checking")
+
+      find("a[href='#{admin_claim_path(claim_to_reject)}']").click
+      choose "Reject"
+      fill_in "Decision notes", with: "TRN doesn't exist"
+      perform_enqueued_jobs { click_on "Submit" }
+
+      expect(claim_to_reject.check.checked_by).to eq(user_id)
+      expect(claim_to_reject.check.notes).to eq("TRN doesn't exist")
+
+      expect(page).to have_content("Claim has been rejected successfully")
+      expect(page).to_not have_content(claim_to_reject.reference)
+
+      mail = ActionMailer::Base.deliveries.last
+
+      expect(mail.subject).to match("been rejected")
+      expect(mail.to).to eq([claim_to_reject.email_address])
+      expect(mail.body.raw_source).to match("been rejected.")
+    end
+
+    scenario "User can see existing check details" do
+      claim_with_check = create(:claim, :submitted, check: build(:check, result: :approved, notes: "Everything matches"))
+      visit admin_claim_path(claim_with_check)
+
+      expect(page).not_to have_button("Submit")
+      expect(page).to have_content("Claim decision")
+      expect(page).to have_content("Result Approved")
+      expect(page).to have_content("Notes\nEverything matches")
```

Comment:
I'm not sure we need to check for the presence of the text in the `dt` here, it tripped me up a bit. I think we should either just check for the presence of the text we added in the note, or do some more complex checks against the HTML (which may be overkill, and likely to make the tests brittle)